internal/disk/disk.go

```diff
@@ -0,0 +1,64 @@
+package disk
+
+import (
+	"gobar/internal/blockutils"
+	"gobar/internal/clickutils"
+	"gobar/internal/log"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+// GetDisk returns a block containing the remaining space for the home partition
+func GetDisk(timeout time.Duration, blockCh chan<- *blockutils.Block) {
+	const hddRune = '\uf51f'
+	var diskSpace string
+	diskBlock := blockutils.Block{
+		Name:        blockutils.DiskName,
+		Border:      blockutils.Red,
+		BorderLeft:  0,
+		BorderRight: 0,
+		BorderTop:   0,
+		Urgent:      false,
+		FullText:    "",
+	}
+
+	for {
+		cmd := exec.Command("df", "-h")
```

Comment:
getting output like this gives me shivers
you'll get more mileage out of a program if you can find the raw data in the filesystem

Reply:
I agree. I'll use `syscall.Statfs` for this in another PR